packages/consumption/src/consumption/ConsumptionCoreErrors.ts

```diff
@@ -266,6 +266,14 @@ class Attributes {
     public setDefaultRepositoryAttributesIsDisabled() {
         return new CoreError("error.consumption.attributes.setDefaultRepositoryAttributesIsDisabled", "Setting default RepositoryAttributes is disabled for this Account.");
     }
+
+    public invalidTag(tag: string): ApplicationError {
+        return new ApplicationError("error.consumption.attributes.invalidTag", `The tag '${tag}' is invalid.`);
+    }
+
+    public invalidTags(tags: string[]): ApplicationError {
+        return new ApplicationError("error.consumption.attributes.invalidTags", `The tags '${tags.join(", ")}' is invalid.`);
```

Comment:
```suggestion
        return new ApplicationError("error.consumption.attributes.invalidTags", `The tags '${tags.join(", ")}' are invalid.`);
```
Also, I would state for which value type they are invalid.

Reply:
also `'${tags.join("', '")}'`